addons/mail/static/src/core/common/thread_model.js

```diff
@@ -87,6 +87,19 @@ export class Thread extends Record {
         onDelete: (r) => r.delete(),
         sort: (m1, m2) => m1.id - m2.id,
     });
+    membersTyping = Record.many("ChannelMember", { inverse: "threadAsTyping" });
+    membersTypingOther = Record.many("ChannelMember", {
+        /** @this {import("models").Thread} */
+        compute() {
+            return this.membersTyping.filter((member) => !member.persona?.eq(this._store.self));
+        },
+    });
+    hasMembersTypingOther = Record.attr(false, {
```

Comment:
These field names are confusing to me.
I don't like this "prefix" notation, they really don't read well.